docs/tutorials/getting-started/connect-to-data.md

```diff
@@ -53,9 +53,16 @@ When you completed those last few steps, you told Great Expectations that:
 
 Based on that information, the CLI added the following entry into your ```great_expectations.yml``` file, under the datasources header:
 
-```python file=../../../tests/integration/docusaurus/tutorials/getting-started/getting_started.py#L16-L35
+```yaml file=../../../tests/integration/docusaurus/tutorials/getting-started/getting_started.py#L17-L34
 ```
 
+:::note What does the configuration contain?
+
+- `ExecutionEngine` : The `ExecutionEngine` provides backend-specific computing resources that are used to read-in and perform validation on data.  For more information on `ExecutionEngines`, please refer to the following [Core Concepts document on `ExecutionEngines`](/docs/reference/execution-engine)
```

Comment:
Personal preference: I would only include backticks for the first instance of a term in the definition (and you may want to **bold** this instance of using backticks. If you decide not to change this, I would still remove the backticks from the term ExecutionEngines within the link at the end of this line (it looks a little strange).

Reply:
I totally agree with your point on **bold**, and removing the backticks from the link, and have adjusted the doc accordingly.  

Regarding the rest of the backticks (I think there are maybe 2 that are not the first instance of a term), I'm honestly a bit torn....  I tried playing around with it a bit, and only having *some* of the terms backtick-ed made it seem like we had "missed" the formatting on some of the docs. Would like to keep them for now, and perhaps revisit if they become too much of an eyesore.

Reply:
adding a short definition of `DataConnectors` for consistency too